.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,14 @@
-For plugin updates, please indicate which repos this should be built into:
+If your package needs to be released within one or more release streams, and/or distributions, please open PRs to each of those branches respectively. The easiest way to do this is to make the initial commit for the mainline branch (e.g. rpm/develop or deb/develop) and then cherry pick the commit hash onto each subsequent branch.
 
-* [x] Nightly
-* [ ] 1.20
-* [ ] 1.19
-* [ ] 1.18
+Example:
+
+    git checkout -b foreman-tasks-1.0.1
+    Make changes
+    git commit -a -m 'Release foreman-tasks-1.0.1'
+    COMMIT=`git rev-parse HEAD`
+    git checkout rpm/1.20
+    git checkout -b foreman-tasks-1.0.1-1.20
```

Comment:
Can be done at once in `git checkout -b foreman-tasks-1.0.1-1.20 rpm/1.20`. I do wonder about deb vs rpm. Also that it needs to be done for every branch.

Reply:
Should I put a `deb` example as well? Yea, the pain point is how many updates one has to make. Automation for users would be great here, but in the meantime.